systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -286,6 +290,18 @@ void testMirrorMaker2Metrics() {
         assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 1));
     }
 
+    @Test
+    void testKafkaBridgeMetrics() {
+        kafkaBridgeMetricsData = MetricsUtils.collectKafkaBridgePodMetrics(BRIDGE_CLUSTER);
+        assertThat("Collected KafkaBridge metrics are empty", kafkaBridgeMetricsData.size() > 0);
+        assertThat("Collected KafkaBridge metrics doesn't contains vertx metrics", kafkaBridgeMetricsData.values().toString().contains("vertx"));
+        assertThat("Collected KafkaBridge metrics doesn't contains jvm metrics", kafkaBridgeMetricsData.values().toString().contains("jvm"));
```

Comment:
Should we have similar check for Kafka Producer and Kafka Consumer metrics?

Reply:
Agreed

Reply:
I was able to see those metrics in grafana, but when I fetch metrics from `bridge:8080/metrics` I can see only this https://gist.github.com/Frawless/4d04010dd2e9d7dfbd3276f1ccf8f2a1, do I need update rules in CR as we have it for Kafka for example?

Reply:
Yes you are right to see kafka consumer and producer metrics you need to start consumer/producer applications otherwise JMX doesn't collect/expose any metrics.

Reply:
Ah make sense. I will add it to the test. Thanks